kubernetes/cmsweb/services/reqmgr2ms-output.yaml

```diff
@@ -0,0 +1,117 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: reqmgr2ms-o
+  namespace: dmwm
+spec:
+  selector:
+    app: reqmgr2ms-o
+  ports:
+    - port: 8248
```

Comment:
This port is wrong, see: https://github.com/dmwm/deployment/blob/master/reqmgr2ms/config-output.py#L36
same applies to other lines in the same file.

Reply:
> @muhammadimranfarooqi thanks for making these changes.
> Given that this PR has been merged without my review, can you please make a new PR with the necessary changes?
> 
> I'd also rather update those names, like:
> 
> * `reqmgr2ms-m` to `ms-monitor`
> * `reqmgr2ms-t` to `ms-transferor`
> * `reqmgr2ms-o` to `ms-output`
> 
> And for my own education, please let me know where these naming are expected to be seen (it can't be a name just to be put in the yaml file...)

@amaltaro 

Ok. Yes, these are not only the names. However you can get these for services, deployments and configmaps. 

For services, you can get like this 

```
[mimran@lxplus765]~/CMSKubernetes-testbed/CMSKubernetes/kubernetes/cmsweb% kubectl get services -n dmwm
NAME            TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)              AGE
ms-monitor      ClusterIP   10.254.225.125   <none>        8248/TCP,18248/TCP   8m8s
ms-output       ClusterIP   10.254.99.66     <none>        8245/TCP,18248/TCP   8m7s
ms-transferor   ClusterIP   10.254.223.215   <none>        8247/TCP,18248/TCP   8m6s
reqmgr2         ClusterIP   10.254.240.234   <none>        8246/TCP,18240/TCP   94m
reqmon          ClusterIP   10.254.138.148   <none>        8249/TCP,18249/TCP   94m
workqueue       ClusterIP   10.254.123.112   <none>        8240/TCP             94m
```

For deployements, 

```
[mimran@lxplus765]~/CMSKubernetes-testbed/CMSKubernetes/kubernetes/cmsweb% kubectl get pods -n dmwm
NAME                             READY   STATUS      RESTARTS   AGE
cron-proxy-1590969900-lsbw7      0/1     Completed   0          2d9h
cron-proxy-1591056300-9cxjl      0/1     Completed   0          33h
cron-proxy-1591142700-z27wd      0/1     Completed   0          9h
ms-monitor-544bdfdbd6-g29lx      1/1     Running     0          9m7s
ms-output-78c976b5b4-wwms4       1/1     Running     0          9m6s
ms-transferor-6d85d8f79b-bh7zh   1/1     Running     0          9m5s
reqmgr2-858884c5b5-856zh         1/1     Running     0          95m
reqmgr2-858884c5b5-cwkm6         1/1     Running     0          95m
reqmgr2-858884c5b5-j8gq7         1/1     Running     0          95m
reqmgr2-858884c5b5-lmmfm         1/1     Running     0          95m
reqmgr2-858884c5b5-xxq5f         1/1     Running     0          95m
reqmgr2-tasks-7fc4db96c8-78kzf   1/1     Running     0          95m
reqmon-7c47b4f5d7-8mwdv          1/1     Running     0          95m
reqmon-7c47b4f5d7-j7wjr          1/1     Running     0          95m
reqmon-7c47b4f5d7-n8f8m          1/1     Running     0          95m
reqmon-7c47b4f5d7-r6qvq          1/1     Running     0          95m
reqmon-7c47b4f5d7-zbbnh          1/1     Running     0          95m
reqmon-tasks-766f8d746d-qk949    1/1     Running     0          95m
workqueue-6fc5b5c4d5-rmfqs       1/1     Running     0          95m

```

For configmaps

```
[mimran@lxplus765]~/CMSKubernetes-testbed/CMSKubernetes/kubernetes/cmsweb% kubectl get configmaps -n dmwm
NAME                 DATA   AGE
ms-monitor           1      9m31s
ms-output            1      9m30s
ms-transferor        1      9m29s
reqmgr2              1      96m
reqmgr2-tasks        1      96m
reqmon               1      95m
reqmon-tasks         1      95m
workqueue            1      95m
```